js/nextperson.js

```diff
@@ -0,0 +1,75 @@
+(() => {
+    let index = 0;
+
+    const peronName = document.getElementById('next-people__name');
+    const peronHeight = document.getElementById('next-people__height');
+    const peronHairColor = document.getElementById('next-people__hair');
+    const peronSkinColor = document.getElementById('next-people__skin');
+    const peronBirthYear = document.getElementById('next-people__birth');
+    const peronGender = document.getElementById('next-people__gender');
+
+    const nextPersonBtn = document.getElementById('next');
+
+    const swapiPeopleAPI1 = (index = 1) => `https://swapi.co/api/people/${index}/?format=json`;
```

Comment:
You dont need **swapiPeopleAPI1** like 6 times, you can write one function **swapiPeopleAPI** which takes index